test/scales/scaleTests.ts

```diff
@@ -291,6 +291,65 @@ describe("Scales", () => {
       assert.equal(scale.scale("a"), "#ff0000");
       assert.equal(scale.scale("b"), "#0000ff");
     });
+
+    it("accepts CSS specified colors", () => {
+
+      var style = d3.select("body").append("style");
+      style.html(".plottable-colors-0 {background-color: #ff0000 !important; }");
+
+      var scale = new Plottable.Scale.Color();
+      assert.strictEqual(scale.range()[0], "#ff0000", "User has specified red color for first color scale color");
+      assert.strictEqual(scale.range()[1], "#fd373e", "The second color of the color scale should be the same");
+
+      style.remove();
```

Comment:
Minor thing; we should move this line (and subsequent `maliciousStyle.remove()` lines) to immediately after the test scale is created. That way, if any `assert` statements fail, the `<style>` element doesn't remain stranded on the page, affecting all other tests.